vignettes/preprocessing-data.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "Pre-processing data"
+author: "NEST CoreDev"
+date: "2022-07-26"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Pre-processing data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+"Data" preprocessing refers to the code which precedes the `teal` app initialization in `app.R`, i.e.:
```

Comment:
We should probably tweak this vignette as it refers to teal apps but we're in teal.data (we have an eval FALSE on line 21)? Other lines w.g. line 17 don't make sense in the teal.data context

Reply:
Updated.